index.html

```diff
@@ -210,29 +210,23 @@ <h3>Andrei Stoean</h3>
 							<div class="custom-card">
 								<div class="avatar-wrapper">
 									<img
-										src="https://media.istockphoto.com/vectors/thumbnail-image-vector-graphic-vector-id1147544807?k=6&m=1147544807&s=612x612&w=0&h=8CXEtGfDlt7oFx7UyEZClHojvDjZR91U-mAU8UlFF4Y="
+										src="fotografie Robert 2..jpeg"
 										alt=""
```

Comment:
1. E foarte bine ca ai pus poza local, insa te sfătuiesc sa o muți in folderul assets, și sa păstrezi doar una singura. 
2. Și denumirea imaginii ar fi bine sa nu conțină spații, adică patreaza doar o poza: fotografie-robert.jpeg.

Dupa ce faci schimbarile astea si src-ul imaginii ar trebui modificat in: <img src="assets/fotografie-robert.jpeg" alt="Robert" class="avatar"/>